README.md

```diff
@@ -30,6 +31,12 @@ foreach ( $walker as $issue ) {
     var_dump($issue);
     // Send custom notification here.
 }
+
+// generic API call
```

Comment:
Please format example so that it will fit on single screen without wrapping. The multi-line function call will do just fine.